src/main/java/TurnUseCases/MovePlayerUseCase/MovePlayerUseCase.java

```diff
@@ -0,0 +1,150 @@
+package TurnUseCases.MovePlayerUseCase;
+import GameEntities.Player;
+import GameEntities.Tiles.*;
+import GameEntities.Board;
+import TurnUseCases.EndTurnUseCase.EndTurnUseCase;
+import GameEntities.Cards.CardActionResultModel;
+
+/**
+ * MovePlayerUseCase (Class to handle moving the player and all its relevant logic such as passing a tile and landing
+ * actions)
+ * MovePlayerUseCase will handle the cases when the player rolls 3 consecutive doubles or land on "go to jail" and send
+ * them to jail, otherwise, it will be a standard move with it moving the player to their final locations and calling
+ * the tile's .action method
+ */
+public class MovePlayerUseCase implements MovePlayerInputBoundary {
+
+    private MovePlayerOutputBoundary movePlayerOutputBoundary;
+    private Board board;
+    private EndTurnUseCase endTurnUseCase;
+    /**
+     * @param movePlayerOutputBoundary MovePlayerOutputBoundary to handle display
+     * @param board The board the game is operating on
+     * @param endTurnUseCase class to force end a turn if the player is sent to jail
+     */
+    public MovePlayerUseCase(MovePlayerOutputBoundary movePlayerOutputBoundary, Board board,
+                             EndTurnUseCase endTurnUseCase) {
+        this.movePlayerOutputBoundary = movePlayerOutputBoundary;
+        this.board = board;
+        this.endTurnUseCase = endTurnUseCase;
+    }
+
+    @Override
+    public void moveToPosition(Player player, int absolutePosition) {
```

Comment:
Although this method appears functionally adequate, its implementation can be much cleaner as it suffers from some code smells (namely long method, and too many data clumps). After doing some unit tests for the methods, I suggest code refactoring by fragmenting the code and eliminating any redundancies. If there aren't any, then maybe rearrange and focus on concision.